doc/man3/EVP_PKEY_get_default_digest_nid.pod

```diff
@@ -13,7 +13,8 @@ EVP_PKEY_get_default_digest_nid - get default signature digest
 
 The EVP_PKEY_get_default_digest_nid() function sets B<pnid> to the default
 message digest NID for the public key signature operations associated with key
-B<pkey>.
+B<pkey>. Note that some signature algorithms (i.e. Ed25519 and Ed448) do not use
+a digest during signing. In this case B<pnid> will be set to NID_undef.
```

Comment:
If I am not mistaken, up to 1.1.0 returning `NID_undef` as the default md would cause errors in the OpenSSL apps.

If that's the case, and this change is ok for a minor release, an HISTORY entry detailing the change might be required.

Reply:
On further investigation, I am mistaken on this, so nevermind this comment.